sage-ios/SAGE/SAGE/Checkin.swift

```diff
@@ -47,9 +48,9 @@ class Checkin: NSObject {
                 let schoolDictionary = value as! [String: AnyObject]
                 self.school = School(propertyDictionary: schoolDictionary)
             case CheckinConstants.kVerified:
-                self.verified = value as? Bool
+                self.verified = value as! Bool
             case CheckinConstants.kId:
```

Comment:
these should actually be optional because sometimes stuff comes back from the backend with no value
like it'll be like: {"verified": nil} or null or whatever